src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeAktivitetServiceTest.kt

```diff
@@ -98,7 +98,7 @@ class VilkårperiodeAktivitetServiceTest : IntegrationTest() {
 
         @Test
         fun `skal lagre kildeId på aktivitet`() {
-            val behandling = testoppsettService.opprettBehandlingMedFagsak(behandling(), opprettGrunnlagsdata = false)
+            val behandling = testoppsettService.opprettBehandlingMedFagsak(behandling(), opprettGrunnlagsdata = true)
```

Comment:
Må opprette grunnlagsdata for å ikke få feilmelding under uthenting av `grunnlagsdata` i `oppdaterVilkårperiode`